src/Library/Lod/LodReader.h

```diff
@@ -30,9 +30,20 @@ class LodReader final {
     bool exists(const std::string &filename) const;
     Blob read(const std::string &filename);
 
+    const auto begin() const {
+        return _index.front().files.cbegin();
+    }
+
+    const auto end() const {
+        return _index.front().files.cend();
+    }
+
  private:
     bool _isFileCompressed(const LodFile &file);
 
+#ifdef _DEBUG
```

Comment:
Don't bother with `#ifdef _DEBUG`, just save the filename here. It'll come in handy when we rework error handling.